package/MDAnalysis/analysis/psa.py

```diff
@@ -944,31 +1529,170 @@ def plot(self, filename=None, linkage='ward', count_sort=False,
+        for tic in ax_hmap.xaxis.get_minor_ticks():
+            tic.tick1On = tic.tick2On = False
+        for tic in ax_hmap.yaxis.get_minor_ticks():
+            tic.tick1On = tic.tick2On = False
+
+        if filename is not None:
+            head = self.targetdir + self.datadirs['plots']
+            outfile = os.path.join(head, filename)
+            savefig(outfile, dpi=600, bbox_inches='tight')
 
+        return Z, dgram, dist_matrix_clus
 
-    def cluster(self, distArray, method='ward', count_sort=False,
-                distance_sort=False, no_labels=True, color_threshold=4):
+
+    def plot_nearest_neighbors(self, filename=None, idx=0,                      \
+                               labels=('Path 1', 'Path 2'), figsize=6.5,        \
+                               multiplot=False, aspect_ratio=1.75,              \
+                               labelsize=12):
+        """Plot nearest neighbor distances as a function of normalized frame
+        number (mapped to the interval *[0,1]*).
+
+        :Arguments:
+          *filename*
+             string, save figure to *filename* [``None``]
+          *idx*
+             integer, index of path (pair) comparison to plot [``0``]
+          *figsize*
+             float, set the vertical size of plot in inches [``6.5``]
+          *aspect_ratio*
+             float, set the ratio of width to height of the plot [``1.75``]
+          *labelsize*
+             set the font size for colorbar labels; font size for path labels on
+             dendrogram default to 3 points smaller [``12``]
+
+        If *filename* is supplied then the figure is also written to file (the
+        suffix determines the file type, e.g. pdf, png, eps, ...). All other
+        keyword arguments are passed on to :func:`pylab.imshow`.
         """
-            Cluster trajectories.
-
-            :Arguments:
-              *method*
-                string, name of linkage criterion for clustering [``'ward'``]
-              *no_labels*
-                boolean, if ``True`` then do not label dendrogram [``True``]
-              *color_threshold*
-                For brevity, let t be the color_threshold. Colors all the
-                descendent links below a cluster node k the same color if k is
-                the first node below the cut threshold t. All links connecting
-                nodes with distances greater than or equal to the threshold are
-                colored blue. If t is less than or equal to zero, all nodes are
-                colored blue. If color_threshold is None or ‘default’,
-                corresponding with MATLAB(TM) behavior, the threshold is set to
-                0.7*max(Z[:,2]). [``4``]]
-            :Returns:
-              list of indices representing the row-wise order of the objects
-              after clustering
+        from matplotlib.pyplot import figure, savefig, tight_layout, clf
+        import seaborn as sns
```

Comment:
@richardjgowers Got it.

@kain88-de seaborn does change the default matplotlib style, unless one uses `import seaborn.apionly`. This was what I did in the `plot_annotated_heatmap` method, but *not* the `plot_nearest_neighbors` method. I am going to change this so that `plot_nearest_neighbors` behaves the same way as `plot_annotated_heatmap` and doesn't catch users by surprise.